source/jsonnet/england-wales/household/blocks/who-lives-here/primary_person_list_collector.jsonnet

```diff
@@ -48,13 +48,21 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
     },
     definitions: [
       {
-        title: 'What does “usually live” mean?',
+        title: 'What we mean by “usually live”?',
```

Comment:
`?` needs to be removed.

Reply:
Removed, that was my own initiative to put a question mark there, sorry 👍